devlib/module/gem5stats.py

```diff
@@ -103,10 +105,10 @@ def start_periodic_dump(self, delay_ns=0, period_ns=10000000):
             raise ValueError(msg.format(delay_ns, period_ns))
         self.target.execute('m5 dumpresetstats {} {}'.format(delay_ns, period_ns))
     
-    def match(self, keys, rois_labels):
+    def match(self, keys, rois_labels, base_dump=0):
         '''
         Tries to match the list of keys passed as parameter over the statistics
-        dumps covered by selected ROIs since origin. Returns a dict indexed by 
+        dumps covered by selected ROIs since base_dump. Returns a dict indexed by 
```

Comment:
Add double grave accents around `base_dump` to mark it as an identifier (similar in the docstring below).